docs/deploy-helm.md

```
@@ -109,6 +109,35 @@ To deploy End-to-End RAG Server and Ingestor Server, use the following procedure
+           value: "1"
+         - name: NIM_SERVED_MODEL_NAME
+           value: "nvidia/nemotron-3-super-120b-a12b"
+         - name: NIM_ENABLE_CHUNKED_PREFILL
+           value: "1"
+         - name: NCCL_NVLS_ENABLE
+           value: "0"
+         - name: VLLM_USE_FLASHINFER_MOE_FP8
+           value: "0"
+         - name: NIM_PASSTHROUGH_ARGS
+           value: "--max-num-seqs 384"
+   ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)